porting/index.md

```diff
@@ -50,6 +55,8 @@ repo init -u https://github.com/Halium/android -b halium-5.1
 
 #### Step 3: Download the source code
 
+*Note:* the repo tool takes some time to download the sources. You need a little patience here. Also it could make sense to first read the next part about the Android tree configuration, since there you need another repo sync call to download everything.
```

Comment:
Ah ok. Get it. The "Note" still doesn't explain it though. I suggest 
* ideally we would move the local manifest section up before this point. However, that might best be done as a future change after completing this pr
* the second best option in my opinion is to entirely remove the "Also it ... everything" sentence. There is a small chance that a not-yet-knowledgeable person will save a little bit of repo-sync-time from this sentence. There is a big chance that it will confuse the person. Should I do something now? What should I do? If later on, something will not have worked, will it have been due to something between these lines that I should have done. After all the guide does right now not mention a second repo sync. The guide also never calls it "Android tree", so again for a new person, not used to the jargon, there's more confusion than help.